jupyterlab_caip_optimizer/src/service/optimizer.ts

```diff
@@ -18,7 +18,16 @@
 import { ServerConnection } from '@jupyterlab/services';
 
 import { TransportService, handleApiError } from 'gcp_jupyterlab_shared';
-import { Study, MetadataFull, MetadataRequired } from '../types';
+import {
+  Study,
+  MetadataFull,
+  MetadataRequired,
+  Trial,
+  Measurement,
+  Operation,
+  SuggestTrialOperation,
+} from '../types';
+import { createTrialUrl } from '../utils/urls';
 
 // const AI_PLATFORM = 'https://ml.googleapis.com/v1';
```

Comment:
Is this supposed to be used eventually? Should we remove it?

Reply:
I think we should remove this.